_posts/2016-04-11-growing-developers-at-greenfield.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: "Growing Developers at Greenfield"
+description: "Advice for junior developers getting started in their careers"
+category:
+tags: [lessons, advice, career]
+author: tom_chen
+---
+{% include JB/setup %}
+
+![tree]({{ BASE_PATH
+}}/assets/images/growing-developers-at-greenfield/tree.jpeg)
+
+Before I was a developer at Greenfield, I spent several years working in
+academia. After deciding to make the leap into full-time software development
+and completing [Launch Academy](https://www.launchacademy.com/) last year, I was
+anxious to join a company where I could work with and learn from experienced
+developers who were shipping high quality code. When I arrived at Greenfield, I
+was happy to find a team that was not only welcoming and supportive but also
```

Comment:
Interesting. I always thought "was" / "has" / "is" automatically make sentences passive:

```
The passive voice does exist for a reason, however, and its presence is not always to be despised. The passive is particularly useful (even recommended) in two situations:

When it is more important to draw our attention to the person or thing acted upon: The unidentified victim was apparently struck during the early morning hours.
When the actor in the situation is not important: The aurora borealis can be observed in the early morning hours.
```

TIL.